test_scripts/API/AutoCompleteList/012_ATF_SetGlobalProperties_N_AutoCompleteList_StringOutLowerBound.lua

```diff
@@ -0,0 +1,60 @@
+-----------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = false, resultCode = "INVALID_DATA"> to mobile app
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param param to HMI
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList> string in array is out lower bound.
+-- 3. SDL respond <success = false, resultCode = "INVALID_DATA"> to mobile app
```

Comment:
@Aleksandriys, looks like you missed "Expected result"(mandatory in template) in header.
I guess, at least p3."SDL respond <success = false, resultCode = "INVALID_DATA"> to mobile app" should be in expected result.